ArduPlane/quadplane.cpp

```diff
@@ -843,7 +843,7 @@ void QuadPlane::run_esc_calibration(void)
  */
 void QuadPlane::multicopter_attitude_rate_update(float yaw_rate_cds)
 {
-    bool use_multicopter_control = in_vtol_mode() && !tailsitter.in_vtol_transition();
+    bool use_multicopter_control = !use_fw_attitude_controllers();
```

Comment:
This is not the same, we loose the `!tailsitter.in_vtol_transition()`

Reply:
fixed

Reply:
I actually don't think we need this change at all. `multicopter_attitude_rate_update` is not called in pos1. Not that it might not be a valid change, but removing it and the tailsitter change it needed in `use_fw_attitude_controllers` makes this a 1 line change.